lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -429,22 +429,6 @@ sub settings_hash {
     return $settings;
 }
 
-sub deps_hash {
-    my ($self) = @_;
-    return $self->{_deps_hash} if defined $self->{_deps_hash};
-    my @dependency_names = OpenQA::JobDependencies::Constants::display_names;
-    my %parents          = map { $_ => [] } @dependency_names;
-    my %children         = map { $_ => [] } @dependency_names;
-    $self->{_deps_hash} = {parents => \%parents, children => \%children};
-    for my $dep ($self->parents) {
-        push @{$parents{$dep->to_string}}, $dep->parent_job_id;
-    }
-    for my $dep ($self->children) {
-        push @{$children{$dep->to_string}}, $dep->child_job_id;
-    }
-    return $self->{_deps_hash};
-}
-
```

Comment:
removing this is an interesting change. Maybe better suited in a separate commit?